scenario-tests/tests/toolkits/test_image_classifier.py

```diff
@@ -265,10 +265,10 @@ def setUpClass(self):
 
 # TODO: if on skip OS, test negative case
 @unittest.skipIf(_mac_ver() < (10,14), 'VisionFeaturePrint_Screen only supported on macOS 10.14+')
```

Comment:
Probably want to change the message in this `skipIf` decorator too.

Reply:
Oops, missed that, thanks.